coding-the-legislation/40_legislation_evolutions.md

```diff
@@ -1,62 +1,46 @@
 # Legislation evolutions
 
-Openfisca handles legislation changes over time.
+Openfisca handles the fact that the legislation changes over time.
 
 ## Parameter evolution
 
-Many legislation parameters are regularly re-evaluated. 
-In that case, formulas usually don't need to be modified: adding the new parameter value in the parameter file is enough.
+Many legislation parameters are regularly re-evaluated while the variables using them stay the same.
+>Example: the `taxes` parameter can change without altering the code of the `flat_tax_on_salary` variable that uses that parameter.
 
-Let's go back to our [previous example](10_basic_example.md#example-with-legislation-parameters):
+In that case, add the new parameter values and their start dates in the appropriate parameter files.
 
-```py
-class flat_tax_on_salary(Variable):
-    column = FloatCol
-    entity = Person
-    label = u"Individualized and monthly paid tax on salaries"
-    definition_period = MONTH
+### How to update a parameter
 
-    def formula(person, period, legislation):
-        salary = person('salary', period)
+1. Open the file where the parameter is described
 
-        return salary * legislation(period).taxes.salary.rate
+```xml
+<NODE code="taxes">
+    <NODE code='salary'>
+      <CODE code="rate" description="Rate for the flat tax on salaries">
+        <VALUE deb="2016-01-01" valeur="0.25" />
+        <VALUE deb="2015-01-01" valeur="0.20" />
+      </CODE>
+    </NODE>
+</NODE>
 ```
-
-and let's assume we have in one of our parameter files the value of the rate for the past couple of years:
-
+2. Change the parameter with its new start date
```

Comment:
- “Add a new value to this parameter”?
- Add `reference` attributes to all values to show that it should always be done.